readme.html

```diff
@@ -1,29 +1,468 @@
+        by the Splunk SOAR Connector.
+    <table>
+        <tr>
+            <th>SDK NAME</th>
+            <th>VERSION</th>
+        </tr>
+        <tr>
+            <td>illumio</td>
+            <td>1.0.3</td>
+        </tr>
+    </table>
+    </p>
+    <p>
+    <h2>Steps to Configure the Illumio Phantom app's asset</h2>
+    <p>
+        Follow these steps to create an asset for the app in Phantom Platform:
+    <ul>
+        <li>Log in to the Illumio platform.</li>
+        <ul>
+            <li>Once logged in, user's name is shown on top-right corner with dropdown where you find <b>My API Keys</b>
+                option.</li>
+            <li>Please select the <b>My API Keys</b> option.</li>
+            <li>Click on <b>Add</b> button on top-left corner, a dialog box will be shown which accepts name and
+                description.</b></li>
+            <li>Enter unique name for your API key and click on <b>create</b> button to create new API key.</li>
+            <li>On successfully creating a new API key, another dialog box will be shown containing the <b>Key
+                    ID</b> and <b>Secret</b>.</li>
+            <li>Copy the <b>Secret</b> value from the dialog box and paste it into any text editor as backup, as it
+                won't
+                be visible again. Once copied, click on Close.</li>
+            <li>This secret value is the <b>API Secret</b> and <b>Authentication Username</b> value is the <b>API
+                    Key</b> that will be used in the asset.</li>
+            <li>Also store the Base URL and port from the URL.</li>
+            <li>Double click on the selected API key to get the Org ID</li>
+        </ul>
+        <li>Log in to your Phantom platform.</li>
+        <ul>
+            <li>Navigate to the <b>Home</b> dropdown and select <b>Apps</b>.</li>
+            <li>Search the Illumio App from the search box.</li>
+            <li>Click on the <b>CONFIGURE NEW ASSET</b> button.</li>
+            <li>Navigate to the <b>Asset Info</b> tab and enter the Asset name and Asset description.</li>
+            <li>Navigate to the <b>Asset Settings</b> tab.</li>
+            <li>Paste the <b>API Key, API Secret, Hostname, Port, and Org ID</b> from Illumio to its respective
+                configuration parameter.</li>
+            <li>Save the asset.</li>
+            <li>Now, test the connectivity of the Phantom server to the Illumio instance by clicking on the <b>TEST
+                    CONNECTIVITY</b> button.</li>
+        </ul>
+    </ul>
+    </p>
+    <h2>Explanation of the Asset Configuration Parameters</h2>
+    The asset configuration parameters affect 'test connectivity' and some other actions of the application. The
+    parameters related to test connectivity action are Illumio API Key, API Secret, Hostname, Port and Org ID.
+    <ul>
+        <li>
+            <b>API Key: </b> Illumio Authentication Username for asset authorization.
```

Comment:
This is not clear: maybe `The API Key Username for the key created in the previous steps`

Reply:
Done